src/parser.rs

```diff
@@ -198,12 +198,12 @@ impl<'s> Parser<'s> {
             },
             Token::XVar => {
                 self.next_token();
-                Ok(vec![Operation::Variable(|input| input.0)])
-            },
-            Token::YVar => {
-                self.next_token();
-                Ok(vec![Operation::Variable(|input| input.1)])
+                Ok(vec![Operation::Variable(|input| input)])
             },
+            // Token::YVar => {
```

Comment:
How about we completely remove Token::YVar also from the Tokenizer?

Reply:
Soon we are going to implement plotting 3D  functions so I think we should handle both 2D and 3D functions parsing.

Reply:
Ok, fine with me.